Include/cpython/code.h

```diff
@@ -219,6 +201,11 @@ typedef enum _PyCodeLocationInfoKind {
     PY_CODE_LOCATION_INFO_NONE = 15
 } _PyCodeLocationInfoKind;
 
+#define Py_UNSTABLE_CODE_H
+#include "unstable/code.h"
+#undef Py_UNSTABLE_CODE_H
```

Comment:
I would prefer to include it in top-level Include/code.h, as done in Include/pystate.h.